plugin/evm/imports_test.go

```diff
@@ -54,10 +56,10 @@ func TestMustNotImport(t *testing.T) {
 		// Importing these packages configures libevm globally and it is not
 		// possible to do so for both coreth and subnet-evm, where the client may
 		// wish to connect to multiple chains.
-		"plugin/evm/atomic": {"core", "core/vm", "params"},
-		"plugin/evm/client": {"core", "core/vm", "params"},
-		"plugin/evm/config": {"core", "core/vm", "params"},
-		"plugin/evm/header": {"core", "core/vm", "params"},
```

Comment:
If this is pressing, let's keep these two in here.
There might be a reason for these to not import those packages, we should not remove them without investigating in depth their indirect dependencies (and a bit of PITA to do)

Reply:
I've based my changes on this comment:
```
// The following sub-packages of plugin/evm must not import core, core/vm
// so clients (e.g., wallets, e2e tests) can import them without pulling in
// the entire VM logic.
// Importing these packages configures libevm globally and it is not
// possible to do so for both coreth and subnet-evm, where the client may
// wish to connect to multiple chains.
```

I'm pretty sure `config` was required by `client` pkg so that's why it was here. I restored the header pkg although it's a bit weird because it actually imports the `customtypes`